Add warning for partner audience type (#5109)
segmentio/segment-docs: src/connections/destinations/catalog/actions-tiktok-audiences/index.md

You must create an audience segment in your TikTok Advertising account. You can send Engage audiences to an existing audience segment, or create a new audience. Please note the `audience_id` as this is required to send Engage audiences to TikTok. 

> warning ""

> > At the moment, you can only send audience data from Segment to a "Partner audience" type of custom audience in TikTok Ads Manager. TikTok's UI doesn't currently allow the selection of this custom audience type, but this destination's **Create Audience** mapping can create "Partner audience" custom audiences in TikTok. 

Reviewer: ```suggestion
> You can only send audience data from Segment to a "Partner audience" custom audience in TikTok Ads Manager. While TikTok's UI doesn't allow you to select this custom audience type, you can use this destination's **Create Audience** mapping to create "Partner audience" custom audiences in TikTok. 
```
